### PR TITLE
Add Comments to front-end

### PR DESCRIPTION
**Pull request by raysarinas**

### CODE REVIEW

File: front-end/src/components/PostDetailModal.tsx [hunk 1]
```diff
@@ -9,6 +9,10 @@ interface Props {
   isOpen: boolean;
 }
 
+/**
+ * Post Details Modal to show the details of an individual post on a stream
+ * @param props 
+ */
 export default function PostListItem(props:Props) {
```
Comment (jmmxp) [inline front-end/src/components/PostDetailModal.tsx]: I think you're right, this should just be renamed to PostDetailModal. (and change imports too)

**Comment by jmmxp** [comment]
For LoggedInUserContext, you can comment `Allows App to globally access the logged in user after login/registration. App then sends UserLogin through props to child components.`

File: front-end/src/components/PostListItem.tsx [hunk 2]
```diff
@@ -13,6 +13,10 @@ interface Props {
   removeFromFeed: Function;
 }
 
+/**
+ * Post list item component to show a preview for an individual post on a stream
```
Comment (VinnyLuu) [inline front-end/src/components/PostListItem.tsx:18]: I think the PostContentEl is the preview. The comment could be like: "PostListItem component is an individual post list item on a stream" or something like this.

**Review by VinnyLuu — APPROVED** [review]
LGTM made a comment about one of the comments hehe. You can push after addressing this and any other reviews

**Review by jmmxp — APPROVED** [review]
Sorry, I'm trying to get a more succinct comment for what LoggedInUserContext is for. Last change, I swear: 

`Allows App to globally access the logged in user after login/registration. App then sends UserLogin through props to child components.`

**Pull request merged**